tsl/test/sql/cagg_joins.sql.in

```diff
@@ -42,173 +42,317 @@ INSERT INTO devices values (1, 'thermo_1', 'Moscow'), (2, 'thermo_2', 'Berlin'),
 CREATE TABLE devices_dup AS SELECT * FROM devices;
 CREATE VIEW devices_view AS SELECT * FROM devices;
 
---Create a cagg with join between a hypertable and a normal table
--- with equality condition on inner join type and realtime aggregation enabled
-CREATE MATERIALIZED VIEW conditions_summary_daily_realtime
+-- Working cases
+-- Cagg with inner join + realtime  aggregate
+
+CREATE MATERIALIZED VIEW cagg_realtime
 WITH (timescaledb.continuous, timescaledb.materialized_only = FALSE) AS
 SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
    AVG(temperature),
-   MAX(temperature),
-   MIN(temperature),
    name
 FROM conditions, devices
 WHERE conditions.device_id = devices.device_id
-GROUP BY name, bucket;
+GROUP BY name, bucket
```

Comment:
```suggestion
GROUP BY bucket, name
```
Any special reason to don't group first by the time bucket?

Reply:
Just to try different orders